scripts/install_protostar_rust.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
I think we should expose both, separate installation and installation of all bindings at once.

We should be able to install them separately if we decide to focus on debugging one thing, then it is good to be able to install single bindings quickly.

We should be also able to install them all together because it is simple - this is for devs who just want to install them and move forward.

Reply:
[allow to install all bindings at once](https://github.com/software-mansion/protostar/pull/1881/commits/781d6b6b7f798353fcab2406ac3dc36cd168fe83)

Reply:
Can you update README about the bindings?

Reply:
they were updated in https://github.com/software-mansion/protostar/pull/1918/commits/c5a81ff78521f96b9d495ee15305bbad77a37de3#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
or did I miss anything?